### PR TITLE
docs: Update documentation post curriculum migration (PT-184605688)

### PR DESCRIPTION
**Pull request by emcelroy**
PT Story: https://www.pivotaltracker.com/story/show/184605688

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,10 +4,31 @@ This is a repository for CLUE curriculum unit and teacher guide data. The unit a
...
+
+Use the problem-section-template.json file as a starting place to build a problem section's content.json file. To populate the `tiles` array with the tiles that should appear in the section, first create the tiles in CLUE, then use CLUE's export/copy-to-clipboard feature to gather the tile's JSON, and then add it to the the `tiles` array.
+
+The unit's root content.json file references the separate problem section files using relative paths. These paths are used by CLUE to import the problem sections when loading the unit content.
 
 ## Adding a Teacher Guide
 
-In an existing unit directory, add a new file named `[unit-code]-teacher-guide.json`. The teacher guide file name for Stretching and Shrinking, for example, is sas-teacher-guide.json.
+In an existing unit directory, add a subdirectory named `teacher-guide`. The teacher guide should have it's own content.json file added to that subdirectory. If the teacher guide contains problem sections, they should go into separate files in the teacher-guide directory, following the same pattern that the main unit uses.
```
Comment (tealefristoe) [inline README.md:34]:
```suggestion
In an existing unit directory, add a subdirectory named `teacher-guide`. The teacher guide should have its own content.json file added to that subdirectory. If the teacher guide contains problem sections, they should go into separate files in the teacher-guide directory, following the same pattern that the main unit uses.
```

File: README.md [hunk 2]
```diff
@@ -4,10 +4,31 @@ This is a repository for CLUE curriculum unit and teacher guide data. The unit a
...
+---------- content.json
+-------- section-2-type
+---------- content.json
+------ problem-2
+-------- section-1-type
+---------- content.json
+
+Note that each problem section's directory name should match the section's type. So if the problem section's type is `introduction`, its directory name should be `introduction`.
+
+Use the problem-section-template.json file as a starting place to build a problem section's content.json file. To populate the `tiles` array with the tiles that should appear in the section, first create the tiles in CLUE, then use CLUE's export/copy-to-clipboard feature to gather the tile's JSON, and then add it to the the `tiles` array.
```
Comment (tealefristoe) [inline README.md:28]: I'd include the keystrokes to export here: cmd+option+e for a single tile, cmd+shift+s to copy the whole document, cmd+option+shift+s to export the whole document as sections.

**Review by tealefristoe — APPROVED** [review]
Looks good!

**Review by scytacki — APPROVED** [review]
Looks good to me. I also agree with Teale's comments.

File: README.md (hunk 2 above)
Comment (emcelroy) [reply README.md:28]: Good idea. Done.

**Pull request merged**